app/default_overrides.go

```diff
@@ -28,6 +28,14 @@ import (
 	coretypes "github.com/tendermint/tendermint/types"
 )
 
+const (
+	// oneGigabit 1Gbit == 1 billion bits
+	oneGigabit = 1_000_000_000
```

Comment:
iirc, this was the bandwidth reccomendations for a consensus node (side note this should be gigabyte I think, since we're converting to bit below)

I could be wrong though, in which case we should update the default bandwidth expectation below